pegasus/tools/visualization.py

```diff
@@ -54,7 +57,11 @@ def calc_tsne(
     if fftw3_loc is None:
         raise Exception("Please install 'fftw3' first to use the FIt-SNE feature!")
 
-    from fitsne import FItSNE
+    try:
+        from fitsne import FItSNE
+    except PackageNotFoundError:
+        print("FItSNE package is not installed")
+        exit()
```

Comment:
Not needed because this is within the `except` block.

Reply:
yes but it we catch the exception without doing anything, the script would continue its execution..

Reply:
Ah, I remember that you have mentioned it before. Thank you for the reminder!

To make the print statement more bold, you may want to add "Exception" ahead, i.e.
```
print("Exception: FItSNE package is not installed, please install fitsne: pip install fitsne")
```